script-library/container-features/src/devcontainer-features.json

```diff
@@ -1121,6 +1121,24 @@
 				"ruby-rails-postgres",
 				"python-3-postgres"
 			]
+		},
+		{
+			"id": "jupyterlab",
+			"name": "Jupyter Lab",
+			"documentationURL": "https://github.com/microsoft/vscode-dev-containers/blob/main/script-library/docs/jupyterlab.md",
+			"options": {
+				"version": {
+					"type": "string",
+					"proposals": ["latest", "3.6.2"],
+					"default": "latest",
+					"description": "Select or enter a jupyterlab version."
+				}
+			},
+			"buildArg": "_VSC_INSTALL_JUPYTERLAB",
```

Comment:
`buildArg` is actually a [legacy attritbute](https://github.com/microsoft/vscode-remote-containers/blob/main/src/configuration/containerFeaturesConfiguration.ts#L22) now, we auto generate the build environment variables programmatically via the `id` (for consistency)

We fall back to buildArg right now, but would be nice to follow the programatic `_BUILD_ARG_${idSafe}=true` for new features.  

We also need to include this somewhere in the `feature-scripts.env`, or it won't be installed by the [`install.sh`](https://github.com/microsoft/vscode-dev-containers/blob/main/script-library/container-features/src/install.sh)script (Which is the only thing implicitly called by the tooling to install).   I'd suggest placing it underneath the python scripts, since this file actually enforces install order (although we eventually want to move that logic into the tooling itself to simplify things (https://github.com/microsoft/dev-container-spec/issues/16)

Reply:
@joshspicer When I try to follow the pattern from the example template repo and do:

```sh
if [ ! -z ${_BUILD_ARG_JUPYTERLAB} ]; then
   .. install
fi
```

It doesn't seem to work. But everything is working fine if I do the install without checking for that. It seems like it should be getting that from the id attribute which is set to `jupyterlab`. Is that not enough?